proto/zitadel/user/v2beta/user_service.proto

```diff
@@ -655,6 +655,28 @@ service UserService {
     };
   }
 
+  rpc RemoveTOTP (RemoveTOTPRequest) returns (RemoveTOTPResponse) {
+    option (google.api.http) = {
+      delete: "/v2beta/users/{user_id}/totp"
+    };
+
+    option (zitadel.protoc_gen_zitadel.v2.options) = {
+      auth_option: {
+        permission: "authenticated"
+      }
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Remove TOTP generator from a user";
+      description: "Remove the configured TOTP generator of the authenticated user. As only one TOTP generator per user is allowed, the user will not have TOTP as a second-factor afterward."
```

Comment:
since an admin can use this endpoint to remove the TOTP factor from a user the "authenticated user" part is wrong
(also wrong in the other remove OTP methods)

Reply:
replaced the "authenticated user" part on all 3 remove endpoints